src/main/java/org/dependencytrack/notification/publisher/AbstractWebhookPublisher.java

```diff
@@ -33,7 +33,7 @@
 public abstract class AbstractWebhookPublisher implements Publisher {
 
     public void publish(final String publisherName, final PebbleTemplate template, final Notification notification, final JsonObject config) {
-        final Logger logger = Logger.getLogger(this.getClass());
+        final Logger logger = Logger.getLogger(AbstractWebhookPublisher.class);
```

Comment:
What's the reason for this change? The logger should refer to the class extending `AbstractWebHookPublisher` (e.g. `JiraPublisher`), not the abstract class itself.

Reply:
You're right, I think it was a local change that I forgot to remove from my commit. I'll revert to the previous behaviour.